### DISCUSSION TITLE
any way to get mouse events other than on_click?

### DISCUSSION THREAD

**Discussion by toppk** [flet-dev/flet#1667] [Ideas]
I have an existing ui that has specific semantics for a textfield, which is that mouse wheel add and subtract values from the text field as does click and right click.   It seems like you don't get more than the basic events from flet, and even looking at the page.on_event, it seems like scroll events are ignored/discarded earlier. 

I think flutter has this concept of listeners. that can be attached anywhere to get "pointer" events.  I'm curious if that can be exposed in flet?

**Comment by ndonkoHenri** [comment]
Try the [`GestureDetector`](https://flet.dev/docs/controls/gesturedetector).
